### PR TITLE
Migrate cloudwatchlogs* modules and tests

### PR DESCRIPTION
**Pull request by alinabuzachis**
Depends-On: https://github.com/ansible/zuul-config/pull/443

Migrate cloudwatchlogs* modules and tests

### CODE REVIEW

File: tests/integration/targets/cloudwatchlogs/tasks/create-delete-tags.yml [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+      New Key with Spaces: Updated Value with spaces
+      NewCamelCaseKey: CamelCaseValue
+      newPascalCaseKey: pascalCaseValue
+      new_snake_case_key: snake_case_value
+  # Mandatory settings
+  module_defaults:
+    community.aws.cloudwatchlogs_log_group:
+      state: present
+      log_group_name: '{{ log_group_name }}'
+    community.aws.cloudwatchlogs_log_group_info:
```
Comment (gravesm) [inline tests/integration/targets/cloudwatchlogs/tasks/create-delete-tags.yml:35]: I believe these tests are failing because these need to be `amazon.aws`.

Comment (alinabuzachis) [reply tests/integration/targets/cloudwatchlogs/tasks/create-delete-tags.yml]: done, thanks!

**Review by gravesm — APPROVED** [review]

**Comment by gravesm** [comment]
regate

**Review by abikouo — APPROVED** [review]

**Comment by alinabuzachis** [comment]
regate

**Comment by goneri** [comment]
regate

**Pull request merged**